Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [workspace]
+resolver = "2"
```

Comment:
Isn't this the default with edition 2021?

Reply:
Thought so too.

Reply:
Resolver is per workspace. Edition is per crate. If you have a workspace the resolver needs to be set there.

Reply:
TIL